R/S3_print.R

```diff
@@ -89,6 +101,7 @@ print.summary.lm_robust <- function(x,
 #' @export
 print.summary.iv_robust <- function(x,
                                     digits = max(3L, getOption("digits") - 3L),
+                                    signif.stars = getOption("show.signif.stars"),
```

Comment:
thread through signif.stars to print_summary_lm_like ?

Reply:
Yes!